sdk/storage/azure-storage-file-share/azure/storage/fileshare/_lease.py

```diff
@@ -90,22 +105,60 @@ def acquire(self, **kwargs):
         self.etag = response.get('etag')  # type: str
 
     @distributed_trace
-    def release(self, **kwargs):
+    def renew(self, **kwargs):
         # type: (Any) -> None
-        """Releases the lease. The lease may be released if the lease ID specified on the request matches
-        that associated with the file. Releasing the lease allows another client to immediately acquire the lease
-        for the file as soon as the release is complete.
+        """Renews the share lease.
+
+        The share lease can be renewed if the lease ID specified in the
+        lease client matches that associated with the share. Note that
+        the lease may be renewed even if it has expired as long as the share
+        has not been leased again since the expiration of that lease. When you
+        renew a lease, the lease duration clock resets.
+
+        :keyword str etag:
+            An ETag value, or the wildcard character (*). Used to check if the resource has changed,
+            and act according to the condition specified by the `match_condition` parameter.
+        :keyword ~azure.core.MatchConditions match_condition:
+            The match condition to use upon the etag.
+        :keyword str if_tags_match_condition
```

Comment:
check these parameters doc and remove the inaccurate ones.

Reply:
Yep, going through their docstrings at the moment since I learned about versioning in docstrings